### PR TITLE
Adapting EstimateScatter into a data corrections computation script

### PR DESCRIPTION
**Pull request by robbietuk**
Addresses #70 

Repurpose the EstimateScatter script to compute all correction factors in a single script. From before, the scatter estimation script computed 80% of all the correction factors as most are needed for the estimation. Everything except randoms computation from delays. This PR will adapt `EstimateScatter.sh` to compute all data corrections. 

Major changes from `EstimateScatter.sh`
 - Two files are output (if `cleanup=1`): the Multiplicative Factors and the Additive Sinogram. Both of these files are named as per script arguments.
 - This `ComputePoissonDataCorrections.sh` calls the `EstimateRandomsFromDelayed.sh` and so a delayed sinogram is required instead of the randoms sinogram. `EstimateRandomsFromDelayed.sh` has recieved light changes.
 - Updated the documentation.

### CODE REVIEW

**Comment by robbietuk** [comment]
Testing shows promise, I am able to compute data corrections easily from my data. I am happy to merge once I recheck it tomorrow.

File: VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+## Gets the scatter par(dir) from the STIR install path (STIR examples are installed with latest versions of STIR)
+STIR_install_dir=$(dirname $(dirname $(command -v estimate_scatter)))
+scatter_pardir="$STIR_install_dir/share/doc/stir-5.0/examples/samples/scatter_estimation_par_files"
+scatter_par="${scatter_pardir}/scatter_estimation.par"
+
+
+## Estimate the randoms from delays
+echo "====="
+echo "Computing contribution due to randoms from DelayedData"
+sh SubScripts/EstimateRandomsFromDelayed.sh ${randoms3d} ${DelayedData}
```
Comment (KrisThielemans) [inline VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh:104]: As that script uses `trap "echo ERROR in $0" ERR` it is actually a `bash` script, so use `bash` here and change the first line in the script.

(In some systems, `sh` is the same as `bash`, but not all)

File: VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh [hunk 2]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+## AUTHOR: Robert Twyman
+## AUTHOR: Kris Thielemans
+## Copyright (C) 2021 University College London
```
Comment (KrisThielemans) [inline VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh:5]: presumably you copied lines from previous scripts, so the date should reflect that. start from 2020?

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
looks good, although I didn't go through it line by line.

are you saying that the scatter problem has disappeared with this?

File: VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh (hunk 1 above)
Comment (robbietuk) [reply VoxelisedSimulation/SubScripts/ComputePoissonDataCorrections.sh:104]: Agreed, I should probably remove these at some point to make it general with shell

**Comment by robbietuk** [comment]
> are you saying that the scatter problem has disappeared with this?

Unfortunely not, that issue remains with this script for the XCAT data. This script simply combines all the data corrections into one.

**Pull request merged**